### PR TITLE
zh-cn: init translation of pwa secure conn

### PR DESCRIPTION
**Pull request by hanyujie2002**
### Description
same as title

### Motivation
ditto

### Additional details
None

### Related issues and pull requests

#15548

Depends on: #15842

### CODE REVIEW

File: files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: 周期跟踪器：安全连接
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:2]: 翻译未统一

File: files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+有几个 {{glossary("IDE")}} 扩展和特定编程语言的包可以通过单击或终端命令启动开发环境。你甚至可以启动多个本地服务器，每个使用不同的端口号。
+
+你可以使用 [VSCode 插件](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#使用文本编辑器的拓展)运行本地 HTTP 服务器，它可以在单个或不同端口上运行本地服务器。[VSCode](https://code.visualstudio.com/download) IDE 的 [Preview on Web Server 扩展](https://marketplace.visualstudio.com/items?itemName=yuichinukiyama.vscode-preview-server)在编辑器当前打开的目录的根目录下创建一个服务器，默认端口为 `8080`。VS Code 扩展是可配置的。`previewServer.port` 设置项是 web 服务器的端口号。可以编辑和更改扩展的默认设置 `8080`。默认情况下，在浏览器的 URL 栏中输入 [`localhost:8080`](/) 将加载该页面。
+
+请注意，Preview on Web Server 扩展使用 browserSync。当你的开发环境由此扩展启动时，[`localhost:3001`](/) 提供当前服务器环境的概述。
+
+你还可以使用 IntelliJ IDE 创建 [本地服务器](https://www.jetbrains.com/help/idea/creating-local-server-configuration.html)，它搭载了一个[可配置的内置 PHP Web 服务器](https://www.jetbrains.com/help/idea/php-built-in-web-server.html#configuring-built-in-web-server)。
+
+了解如何使用 [Python](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#使用 Python) 或本地服务器端语言（如 PHP）来[设置本地测试服务器](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#在本地运行服务器端语言)。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:46]: 我在[这里](https://github.com/mdn/translated-content/pull/15665#issuecomment-1712505345)提到过了如何确定hash部分，不要再搞错了。现在这样渲染都是有问题的：

image

https://pr15811.content.dev.mdn.mozit.cloud/zh-CN/docs/Web/Progressive_web_apps/Tutorials/CycleTracker/Secure_connection#%E4%BD%BF%E7%94%A8%E7%AB%AF%E5%8F%A3%E5%8F%B7%E7%9A%84_localhost

Comment (hanyujie2002) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:46]: I am so sorry, I will be more careful next time👌

Comment (hanyujie2002) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:46]: On, I remember. That corresponding part of English doc has not been translated, so maybe it is a good choice to update translation of that article first

Comment (yin1999) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:46]: You still haven't updated the [hash](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash) of the url. I mean, the hash here (`#使用 Python`) is not correct, please check the [steps here](https://github.com/mdn/translated-content/pull/15665#issuecomment-1712505345) to learn how to determine this.

Comment (hanyujie2002) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md]: OK now, please check 😄

File: files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+有几个 {{glossary("IDE")}} 扩展和特定编程语言的包可以通过单击或终端命令启动开发环境。你甚至可以启动多个本地服务器，每个使用不同的端口号。
+
+你可以使用 [VSCode 插件](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#使用文本编辑器的拓展)运行本地 HTTP 服务器，它可以在单个或不同端口上运行本地服务器。[VSCode](https://code.visualstudio.com/download) IDE 的 [Preview on Web Server 扩展](https://marketplace.visualstudio.com/items?itemName=yuichinukiyama.vscode-preview-server)在编辑器当前打开的目录的根目录下创建一个服务器，默认端口为 `8080`。VS Code 扩展是可配置的。`previewServer.port` 设置项是 web 服务器的端口号。可以编辑和更改扩展的默认设置 `8080`。默认情况下，在浏览器的 URL 栏中输入 [`localhost:8080`](/) 将加载该页面。
+
+请注意，Preview on Web Server 扩展使用 browserSync。当你的开发环境由此扩展启动时，[`localhost:3001`](/) 提供当前服务器环境的概述。
+
+你还可以使用 IntelliJ IDE 创建 [本地服务器](https://www.jetbrains.com/help/idea/creating-local-server-configuration.html)，它搭载了一个[可配置的内置 PHP Web 服务器](https://www.jetbrains.com/help/idea/php-built-in-web-server.html#configuring-built-in-web-server)。
+
+了解如何使用 [Python](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#使用_Python) 或本地服务器端语言（如 PHP）来[设置本地测试服务器](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#在本地运行服务器端语言)。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:46]: Please lowercase the hash. If you're not familiar with this, I recommend checking the rendered page to determine the actual value of it.

```suggestion
了解如何使用 [Python](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#使用_python) 或本地服务器端语言（如 PHP）来[设置本地测试服务器](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server#在本地运行服务器端语言)。
```

Comment (hanyujie2002) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md]: Apologize for my being careless again ＞﹏＜

Comment (hanyujie2002) [reply files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md]: Thank you for your patience 👍 @yin1999

**Review by yin1999 — APPROVED** [review]
Thanks @hanyujie2002. I've allowed myself to do some improvements on it. Now it looks good to me.

**Pull request merged**